src/Main.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+
+import Sanitize
+import Scraping
+import Screenshot
+
+from Quiz import Quiz
+from argparse import ArgumentParser
+from colorama import Fore, Back, Style
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+def play(num_questions,path_directory,path_screenshot):
+    # Create a new Quiz
+    quiz = Quiz(path_directory)
+    # Create a ThreadPool to parallelize the work from here on
+    pool = ThreadPoolExecutor(max_workers=4)
+
+    # For each question
+    for i in range(1,num_questions+1):
+        # Waiting for user input...
+        try:
+            c = input("Press enter to evaluate a new question, e to exit: ")
+        except KeyboardInterrupt:
+            # If CTRL+C, break
+            print("") # Go to a new line  
+            break
+        # If the user decided to exit, break
+        if c == 'e': break
```

Comment:
puoi gestire il caso di uscita usando il rise KeyboardInterrupt e gestire l'uscita in un unico punto? o magari qualcosa di ancora meglio invece di usare un break